packages/react-i18n/tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+	"extends": "../../tsconfig.base.json",
+	"compilerOptions": {
+		"rootDir": "src",
+		"declarationDir": "build-types"
+	},
+	"references": [
+		{ "path": "../element" },
+		{ "path": "../hooks" },
```

Comment:
The react-i18n package doesn't use the hooks package now

Reply:
Fixed 🙂